scripts/utils.sh

```diff
@@ -277,4 +277,20 @@ function retry() {
     return ${rc}
 }
 
+function get_pod_name() {
+    local pod_label=$1
+    local namespace=$2
+
+    kubectl get pods -n $namespace -l $pod_label -o name
+}
+
+function is_pod_ready() {
+    local pod_name=$1
+    local namespace=$2
+
+    while [ $(kubectl get -n $namespace $pod_name -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}') !=  "True" ]; do
+        sleep 3
+    done
```

Comment:
better to use builtin check instead ( also adding a timeout is good practice, we should set it as a variable) 
```suggestion
 kubectl wait -n $namespace --for=condition=Ready=false --timeout=60s  $pod_name
```

Reply:
also I perfer to check if it's `running` not `Ready`
```suggestion
  kubectl wait -n $namespace --for=jsonpath='{.status.phase}'=Running --timeout=60s  $pod_name
```